js/index.js

```diff
@@ -139,3 +139,69 @@ if (typeof zenscroll != 'undefined') {
   var edgeOffset = 56; // px
   zenscroll.setup(defaultDuration, edgeOffset);
 }
+
+/* added by Tom Kolouch */
```

Comment:
This attribution isn't necessary as Git blame can tell us who did this. ;)

Reply:
OK, just old bad habits :-) will not pass any attribution like this into code in future